package.json

```diff
@@ -136,7 +138,7 @@
     "chai": "^4.3.4",
     "chai-jq": "^0.0.9",
     "clean-webpack-plugin": "^3.0.0",
```

Comment:
```suggestion
    "clean-webpack-plugin": "^4.0.0",
```
Should we have the same version as in the premium plugin?

Reply:
I decided to remove the plugin in favor of using output.clean setting.